admiral/logbook.h

```diff
@@ -43,7 +43,7 @@
 
 #define LB_RPT_LOGGER(MESSG)
 #ifdef LOGGING_TO_RPT
-    #define LB_RPT_LOGGER(MESSG)                diag_log (MESSG)
+    #define LB_RPT_LOGGER(MESSG)                diag_log (text MESSG)
```

Comment:
Macro parameters should be wrapped in parentheses. The inlined expression could be evaluated in the wrong order, becasue of unexpected operator precedence.
What I'm trying to say is do this diag_log text (MESSG).